packages/CanvasIFrameModule/custom/themes/custom.less

```diff
@@ -0,0 +1,3 @@
+.full-width {
+  width: 100%;
+}
```

Comment:
Actually, I'm not sure we want to add this CSS here. This could conflict with existing CSS customizations made to this file (for rebranding, etc.).

We could add CSS as part of Handlebars template in a <style> tag or hardcode the style attribute on the element.